src/integrant/core.cljc

```diff
@@ -24,13 +30,11 @@
 (defn ref
   "Create a reference to a top-level key in a config map."
   [key]
-  {:pre [(valid-config-key? key)]}
```

Comment:
In order for this to work, we have to remove this `:pre`. `ref` and `refset` get evaluated by `read-string`, which is before the `build` function.